crowbar_framework/app/models/api/node.rb

```diff
@@ -171,8 +159,8 @@ def save_node_state(role, state = "upgrading")
     end
 
     def save_error_state(message = "")
-      # FIXME: save the error to global status
       Rails.logger.error(message)
+      raise message
```

Comment:
now that it just logs and raises an exception, the `save_error_state` name is also a bit misleading, as it actually saves in the next method